privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -261,52 +383,76 @@ def perform_prediction(
         predictions = meta_classifier.predict(x=[feature_extraction_target_model])
         return predictions
 
-    @staticmethod
-    def output_attack(predictions_ratios: Dict[float, np.ndarray]) -> string:
+    def output_attack(self, predictions_ratios) -> UserOutputPropertyInferenceAttack:
```

Comment:
Probably mentioned in previous sprints: you might prefer documenting python functions with verbs as a first word, not nouns. Because a function "does" something.

Reply:
Okay, changed it.